.travis.yml

```diff
@@ -1,23 +1,43 @@
 language: android
 jdk: oraclejdk8
 sudo: false
+
 addons:
   apt:
     packages:
     - lynx
+
 android:
   components:
   - platform-tools
   - tools
   - build-tools-28.0.3
   - android-28
+  - android-22
   - extra-android-m2repository
-before_install: chmod +x gradle/buildWithTravis.sh
-script: gradle/buildWithTravis.sh
+  - sys-img-armeabi-v7a-android-22
+
+before_install:
+  - chmod +x gradle/buildWithTravis.sh
```

Comment:
`buildWithTravis` is no longer used, it seems. How is uploading to bintray expected to work?

Reply:
it is not currently working... I don't know if Travis-CI made some change in their API, but for now I'm publishing from my machine